search/grammar_v2_engine.go

```diff
@@ -948,3 +1065,105 @@ func (e *ESEngine) sourceUidByTerm(term string, languages []string) (*string, st
 	}
 	return nil, "", false
 }
+
+func (e *ESEngine) isTermRestricted(term string, languages []string) bool {
```

Comment:
Not like this as you don't tokenize anything here...
We should try tokenize everything in elastic.

Reply:
Add comment